src/email-templates/erasure.js

```diff
@@ -28,3 +26,27 @@ ${data.name}
 `);
   }
 };
+
+const CCPA_TEXT = `CCPA To whom it may concern:
+
+I am writing to request that you erase all my personal information from all your information systems pursuant to Article 17 of the General Data Protection Regulation (GDPR). To the extent that you rely on consent to process my personal data, I withdraw that consent. To the extent that you rely on your 'legitimate interest' to process my personal data, I object to the processing as there are no overriding legitimate grounds.
+
+Please confirm that you have erased my personal information from your systems, and that you have followed up with any controller with whom my information has been shared to ensure that they erase their copy of the data. If you need any further information from me in order to identify me or locate my records in your systems, please let me know as soon as possible. My preferred method of contact is email.
+
+Please note that I do not consent to any personal information which is part of this request, including my email address and name, to be used for any purpose other than fulfilling this request.
+
+If you do not normally deal with these requests, please pass along this letter to your Data Protection Officer. Please note that you have 30 days to comply with this request.
+
+`;
+
+const GDPR_TEXT = `GDPR To whom it may concern:
+
+I am writing to request that you erase all my personal information from all your information systems pursuant to Article 17 of the General Data Protection Regulation (GDPR). To the extent that you rely on consent to process my personal data, I withdraw that consent. To the extent that you rely on your 'legitimate interest' to process my personal data, I object to the processing as there are no overriding legitimate grounds.
+
+Please confirm that you have erased my personal information from your systems, and that you have followed up with any controller with whom my information has been shared to ensure that they erase their copy of the data. If you need any further information from me in order to identify me or locate my records in your systems, please let me know as soon as possible. My preferred method of contact is email.
+
+Please note that I do not consent to any personal information which is part of this request, including my email address and name, to be used for any purpose other than fulfilling this request.
+
+If you do not normally deal with these requests, please pass along this letter to your Data Protection Officer. Please note that you have 30 days to comply with this request.
+
+`;
```

Comment:
Not sure you want to merge this until you have edited the CPPA and GDPR messages?

Reply:
I will.